apps/console/src/features/server-configurations/pages/governance-connectors.tsx

```diff
@@ -89,6 +91,17 @@ export const GovernanceConnectorsPage: FunctionComponent<GovernanceConnectorsPag
         getConnectorCategory(categoryId)
             .then((response: GovernanceConnectorCategoryInterface) => {
 
+                if(!OrganizationUtils.isCurrentOrganizationRoot()) {
+                    response.connectors = 
+                        GovernanceConnectorUtils
+                            .filterGovernanceConnectorCategories(categoryId, response.connectors);
+
+                    // need to throw a proper error here if the response.connetors is null
+                    if (response.connectors.length==0) {
+                        throw Error();
```

Comment:
Can we dispatch an error notification?

Reply:
Isn't this done in the catch section ?

Reply:
We can dispatch an error notifiction to anywhere to let the user know something has gone wrong.